Improve GUI font scaling to better support high-DPI displays (#709):

Nadahar opened the pull request:
@UniversalMediaServer/developers @SubJunk 
In response to #709 I've made the following changes:
- Made font sizes and padding in help files CSS scale with OS font size
- Made the log view/traces scale with OS font size
- Made row height in tables in PluginTab and NavigationShareTab scale with OS font size

Please note that since I don't own a high-DPI display, it's a bit hard for me to test this, but I've tuned my Windows font size to 150% while testing and the results looks reasonable. While I don't say that this makes the GUI properly support font scaling, it is better than today and at least makes it functional. Maybe you can test it @valib, since you use font scaling above 100%.

valib commented:
@Nadahar tested and there are some issues. Font size scaling in the TracesTab works fine. But the Help Page font size is growing more than necessary. See:
![ums1](https://cloud.githubusercontent.com/assets/1472678/10783208/485b87ba-7d56-11e5-98a1-f0f9eef841f9.jpg)
I don't know which part of the code make it but it makes the space between lines much bigger like:
![ums2](https://cloud.githubusercontent.com/assets/1472678/10783244/9d515b00-7d56-11e5-964d-f60a9e830474.jpg)

Nadahar commented:
@valib The font in the help tab is quite big without font scaling as well. I've just kept the same ratio. Without font scaling the "standard" font in Windows is 12 px - and the help page text is 14 px. When I tested at 150% the "standard" font was 18 px. To find the help text font I simply multiply wit 7 and divide with 6. 18*7/6=21. I think the help page font is too large in general, but I haven't touced this. I assumed that 14 was selected by design.

I haven't touched the ```About``` tab - I didn't know there were issues there. Do you use the default font size in UMS, or have you configured some special font size?

valib commented:
@Nadahar I only changed in the WIN 10 the text and application size to 150%

Nadahar commented:
@valib Without my changes, the help page doesn't scale (it stays at 14px font size). What I'm trying to say is that the help page scales as it's intended, but that the font size there is bigger than the rest of UMS altso for 100%. I don't think it looks very nice, but it didn't want to change anything, I just added scaling.

If there's consensus to make the help page font smaller, I'll do that. I'd recomment 12px in 100% - the same as the standard font. That will make the help page font 18px i 150% setting.

Nadahar commented:
@valib I've fixed it. I didn't know that, but it seems that all Swing components share a common ```StyleSheet``` (whether they are HTML or not), and when I manipulated the ```StyleSheet``` for the ```EditorPane``` in ```HelpTab``` I actually manipulated the shared ```StyleSheet```. The reason only ```AboutTab``` was affected was that it was the only one being built after ```HelpTab```. 

I'd also replaced ```pt``` (points) with ```px``` (pixels) in the ```StyleSheet``` since I at some point deducted that Swing didn't really support points and that is simply was an alias for pixels. However I drew that conclusion it was wrong - there definitly is a difference.

Will you test it again now? You must reset your local branch since I've amended the last commit.

valib commented:
@Nadahar tested and it looks good. Great job :+1:

Nadahar commented:
@valib Great, thank you for testing.

@UniversalMediaServer/developers @SubJunk 
Any other objections to a merge?

Nadahar commented:
I've tested this quickly on Centos 7, Ubuntu 12 and OS X 10.8 as well and can't see any issues. Since there has been no objections I'm merging this.
